src/DB_Command.php

```diff
@@ -539,6 +539,9 @@ public function export( $args, $assoc_args ) {
 	 * [--dbpass=<value>]
 	 * : Password to pass to mysql. Defaults to DB_PASSWORD.
 	 *
+	 * [--<field>=<value>]
+	 * : Extra arguments to pass to mysqldump. [Refer to mysqldump docs](https://dev.mysql.com/doc/en/mysqldump.html#mysqldump-option-summary).
```

Comment:
We're not using `mysqldump` here, but rather the `mysql` binary. Comment and link to docs need to be adapted.